accelerator/cli_info.h

```diff
@@ -46,14 +46,15 @@ typedef enum ta_cli_arg_value_e {
   MILESTONE_DEPTH_CLI,
   MWM_CLI,
   SEED_CLI,
-  CACHE,
+  CACHE_ACQUIRE,
```

Comment:
Let's shorten the identifier.

Reply:
Should I name it as `CACHE_ON`?

Reply:
Use `CACHE` since we don't get good idea. Mention the default status about caching within commit message.